CHANGELOG_PENDING.md

```diff
@@ -43,6 +43,7 @@ Special thanks to external contributors on this release:
 - [mempool, rpc] \#7041  Add removeTx operation to the RPC layer. (@tychoish)
 
 ### IMPROVEMENTS
+- [internal/protoio] [#7325](https://github.com/tendermint/tendermint/pull/7325) Optimized MarshalDelimited by inlining the common case and using a sync.Pool in the worst case.
```

Comment:
- [ ] attribute changes to your username
- [ ] use backticks around `sync.Pool` and `MarshalDeliminated`
- [ ] we have a script that links tickets so you can replace the link with `\#7325`

Reply:
Done, thank you for the feedback!